sql-ballerina/Ballerina.toml

```diff
@@ -28,3 +28,9 @@ groupId = "org.hsqldb"
 scope = "testOnly"
 artifactId = "hsqldb"
 version = "2.5.1"
+
+[[platform.java11.dependency]]
+path = "./lib/io-native-@io.native.version@.jar"
+groupId = "org.ballerinalang"
+artifactId = "io-native"
+version = "@io.native.version@"
```

Comment:
why we need to add io native dependency in SQL module?

Reply:
This dependency is needed to cater for the extern function stderr(). If stderr() is going to remove, then this dependency can be removed as well.